frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -126,25 +153,34 @@ export const CreateOCSServiceForm = withHandlePromise<
         }
       >
         <div className="ceph-ocs-install__ocs-service-capacity--dropdown">
-          <OCSStorageClassDropdown onChange={setStorageClass} data-test-id="ocs-dropdown" />
+          <OCSStorageClassDropdown onChange={handleStorageClass} data-test-id="ocs-dropdown" />
         </div>
+        {storageClass?.provisioner === NO_PROVISIONER && (
+          <PVsAvailableCapacity
+            replica={ocsRequestData.spec.storageDeviceSets[0].replica}
+            data-test-id="ceph-ocs-install-pvs-available-capacity"
+            sc={storageClass}
+          />
+        )}
```

Comment:
Note: ternary operator can't be used for `storageClass?.provisioner === NO_PROVISIONER` as some element is one `FormGroup` and some in another.